src/systems/joint_position_controller/JointPositionController.cc

```diff
@@ -63,6 +64,14 @@ class ignition::gazebo::systems::JointPositionControllerPrivate
 
   /// \brief Joint index to be used.
   public: unsigned int jointIndex = 0u;
+
+  /// \brief Operation modes
+  enum OperationMode {
+    PID, ABSOLUTE
```

Comment:
Style nit: put each enum in its own line and give it a `\brief`

Reply:
Resolved in 8ceaed2